doc/current-user-auth.md

```diff
@@ -0,0 +1,57 @@
+# Current User Authorization
+
+**Note**: The feature is experimental, which means it may not work as expected and breaking changes may be introduced
```

Comment:
```suggestion
**Note**: The feature is experimental, which means it may not work as expected, it may cause Grafana to behave in an unexpected way, and breaking changes may be introduced
```

Reply:
I would also add a note admonishing what we've already identified r.e. issues with backend features.